### ISSUE TITLE
ImportError: <module 'mozdownload' from '/env/local/lib/python2.7/site-packages/mozdownload/__init__.pyc'> has no 'cli' attribute

### ISSUE THREAD

**Issue by aleksejrs** [mozilla/mozdownload#301]
```
$ mozdownload -a firefox -t daily --branch=mozilla-central -l en-US
Traceback (most recent call last):
  File "/env/bin/mozdownload", line 9, in <module>
    load_entry_point('mozdownload==1.16', 'console_scripts', 'mozdownload')()
  File "/env/local/lib/python2.7/site-packages/pkg_resources.py", line 356, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/env/local/lib/python2.7/site-packages/pkg_resources.py", line 2476, in load_entry_point
    return ep.load()
  File "/env/local/lib/python2.7/site-packages/pkg_resources.py", line 2195, in load
    raise ImportError("%r has no %r attribute" % (entry, attr))
ImportError: <module 'mozdownload' from '/env/local/lib/python2.7/site-packages/mozdownload/__init__.pyc'> has no 'cli' attribute
```

**Comment by dminor** [comment]
This should be fixed by https://github.com/mozilla/mozdownload/commit/aeee681d587ded7d6e6c9e9e00e9df1b709d0643 but I'm not sure we've pushed out a new release since then.

**Comment by aleksejrs** [comment]
I created that new virtualenv before filing the bug.  Debian jessie.

**Comment by dminor** [comment]
Works for me from a fresh clone of mozdownload.

**Comment by whimboo** [comment]
For reference this issue has been fixed with PR #298. Given that we had no tests for that and that I simply missed to run the CLI manually a broken release went out. :( I'm going to get a new release out by today. Thanks @dminor for fixing it.
